python-package/requirements.txt

```diff
@@ -1,4 +1,5 @@
 cryptography==41.0.3
 PyJWT==2.8.0
 pyOpenSSL==23.2.0
-urllib3==1.26.16
+requests==2.31.0
```

Comment:
> The `requests` library is not marked as a dependency of this package

do we need it here?

Reply:
I suppose it depends on what we use this file for. If it's for development dependencies, then yes; if it's for general dependencies, then I'd say no.

Reply:
Maybe I should rename it to `requirements-dev.txt`.